src/store/actions/resource.js

```diff
@@ -104,13 +104,14 @@ export const loadH5pResource = (activityId) => async (dispatch) => {
 export const loadH5pResourceSettings = (activityId) => resourceService.h5pResourceSettings(activityId);
 export const loadH5pResourceSettingsOpen = (activityId) => resourceService.h5pResourceSettingsOpen(activityId);
 export const loadH5pResourceSettingsShared = (activityId) => resourceService.h5pResourceSettingsShared(activityId);
+export const loadH5pResourceSettingsEmbed = (activityId) => resourceService.h5pResourceSettingsEmbed(activityId);
 
 export const createResourceAction = (
   playlistId,
   editor,
   editorType,
   metadata,
-  projectId,
+  // projectId,
 ) => async (dispatch) => {
   // try {
```

Comment:
Are we using this try catch?

Reply:
nope, this is commented out. I think this is commented out while debugging Maths  H5P by simon

Reply:
Okay. Merging